app/models/concerns/developers/notifications.rb

```diff
@@ -55,5 +59,9 @@ def notify_admins_of_potential_hire
     def send_invisiblize_notification
       InvisiblizeNotification.with(developer: self).deliver_later(user)
     end
+
+    def send_product_feature_notification
+      ProductFeatureNotification.with(developer: self).deliver_later(user)
+    end
```

Comment:
This is only one line, we can move it directly into `#notify_product_feature_update`.

Reply:
Sure, will change it. Was just following same standards as done for other methods like notify_as_stale here.